indico/modules/attachments/controllers/event_package.py

```diff
@@ -41,6 +41,29 @@
 from MaKaC.conference import SubContribution
 
 
+def adjust_path_length(segments):
+    """
+    Shorten the path length to < 260 chars.
+
+    Windows' built-in ZIP tool doesn't like files whose
+    total path exceeds 260 chars. Here we progressively
+    shorten the total until it matches that constraint.
+    """
+    result = []
+    total_len = sum(len(seg) for seg in segments)
```

Comment:
That's why I am checking against 255 instead of 260. But yeah, we could count them already in `total_len` if we wanted.